ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,409 @@
+do it – the important thing is to make sure that everyone knows when a ZEP is
+on the verge of acceptance, and give them a final chance to respond. If
+there’s some special reason to extend this final comment period beyond
+`7 days`, then that’s fine, just say so in the post. You shouldn’t do less
+than `7 days`, because sometimes people are travelling or similar and need
+some time to respond.
+
+There may be a case that a ZEP didn’t attract needed attention towards it, the
+engagement from the community is low, and `7 days` pass by. In this case,
+the author(s) of the ZEP must make necessary efforts to spread the word about
+the ZEP through [Gitter] or using the organisation [discussions] feature of
+GitHub or, if needed, creating an additional PR in the [community] repository.
+In addition, the author(s) should get in touch with the [Zarr Steering Council]
+to prevent the case that the ZEP was accepted due to less participation from the
+community.
+
+If all the above options are exhausted by the author(s), then it’s the
+responsibility of the [Zarr Steering Council] to take the final decision on the
+ZEP.
+
+In general, the goal is to make sure that the community has consensus, not
+provide a rigid policy for people to try to game. When in doubt, err on the
+side of asking for more feedback and looking for opportunities to compromise.
+
+If the final comment period passes without any substantive objections, then the
+ZEP can officially be marked `Accepted`. You should create a follow-up
+discussion thread in zarr-developers/zep repository notifying everyone
+(celebratory emoji optional but encouraged 🎉✨), and then update the ZEP by
+setting its `:Status:` to `Accepted`, and it's `:Resolution:` header to a link
+to your follow-up discussion thread.
+
+If there are substantive objections, then the ZEP remains in `Draft` state,
+discussion continues as normal, and it can be proposed for acceptance again
+later once the objections are resolved.
+
+In unusual cases, the [Zarr Steering Council] may be asked to decide whether a
+controversial ZEP is `Accepted`.
+
+### Maintenance
+
+In general, SPEC ZEPs are no longer modified after they have reached the Final
+state. The changes made to the Zarr’s core specification in the
+[zarr-specs] repository are considered the ultimate reference. However,
+finalised SPEC ZEPS may be updated with minor changes as needed.
+
+Process ZEPs may be updated over time to reflect changes to development
+practices and other details. The precise process followed in these cases will
+depend on the nature and purpose of the ZEP being updated.
+
+## ZEP Format
+
+ZEPs are UTF-8 encoded text files using the
+[reStructureText](http://docutils.sourceforge.net/rst.html) format. Please see
+the [ZEP X - Template and Instructions][template.md] file and the
+[reStructuredTextPrimer](https://www.sphinx-doc.org/en/stable/rest.html) for
+more information.
```

Comment:
Very minor comment, and the rest of this looks great.

This says ZEPs are in rst, but this document and the template are in markdown. These should probably be aligned. As a note, this is repeated multiple times throughout the document.

Would be nice if they aligned on markdown 😉

Reply:
Hi @ivirshup.

ZEP0 documents(files in this PR) will be converted into .rst files in a separate repo named `zep`. The markdown files are here so that everyone can read and review them easily.

I'd be taking care of this ^ after this PR is finalised and there are no more comments/objections.

Reply:
Makes sense.

Wondering if it might then also make sense to keep a copy of the markdown template around so people writing ZEPs can do the same?

Reply:
Yeah, that sounds right. I think we can leave the ZEP 0 files (in .md format) here too.